### PR TITLE
Added embedded FAQ description to README.md

### PR DESCRIPTION
**Pull request by strump**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ faq-menu-title = "{Translation of 'F.A.Q.' to a new language}"
 
 **Limitation**: Each F.A.Q. category has only a name. No description, no icon. Only name 'App', or 'Bookmarks and Tracks', or 'Map Editing', etc.
 
+## Embedded FAQ
+
+A subset of FAQ questions is used to generate static single-page FAQ: [`/faq/embedded-faq/`](https://organicmaps.app/faq/embedded-faq/?lang=en). This page should be used as an embedded offline help page in Organic Maps app on Android and iOS.
```
Comment (biodranik) [inline README.md:101]: Maybe tell that the lang in the url can be replaced to test another language?

**Review by biodranik — COMMENTED** [review]
Thanks! Looks like deployment was triggered for readme change. It would be great to add readme and maybe some other tool files to the deployment ignore list )

File: README.md (hunk 1 above)
Comment (strump) [reply README.md]: Added multilang test links and list of supported languages

**Review by biodranik — APPROVED** [review]
Дякую!

**Pull request merged**